recipe/meta.yaml

```diff
@@ -10,44 +10,49 @@ source:
   sha256: 793359e94eb436b34877b4305570774f9b1bba34c4671238512b80b60456d9ce
 
 build:
+  number: 0
+  # maturin isn't available on s390x
+  skip: true  # [py<39 or s390x]
   script:
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
-    - {{ PYTHON }} -m pip install . -vv
-  number: 1
-  skip: true  # [py<39]
-  skip: true  # [ppc64le and python_impl == 'pypy']
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - maturin >=0.14,<0.15
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
-    - cargo-bundle-licenses
+    - cargo-bundle-licenses  # [win]
+    - maturin                # [win]
```

Comment:
why is Maturin a dependency only on windows? I cannot find the source for this.

Reply:
It failed if `maturin` on `win` is missing in `build` even if we already have it in `host` too

Reply:
If you remove `maturin` from `build` on `win64` you will get an error:
```
  File "C:\b\abs_24u2n4km7j\croot\lazrs-python_1685112840081\test_tmp\run_test.py", line 2, in <module>

    import lazrs

ModuleNotFoundError: No module named 'lazrs'
```
Looks like `conda-build` cannot build a package properly on `win` if `maturin` isn't available in `build`

Reply:
A little comment could help future devs that will update this package.

Reply:
Thanks. I've added a comment